### ISSUE TITLE
Campaign never marked as send with custom segment class

### ISSUE THREAD

**Issue by YannikFirre** [spatie/mailcoach-support#179]
Hello :) 

**PHP**: 7.4.5
**Laravel**: 7.10.3
**Mailcoach**: 2.11.4

When i use a custom Segment class in campaign, the campaign is never marked as send always in progress. 

RM : No issue if the segment return true for all the subscribers. 

Example of Segment Class : 
```php
public function shouldSend(Subscriber $subscriber): bool
{
    if (! empty($this->languages) && ! in_array($subscriber->extra_attributes->language, $this->languages, false)) {
         return false;
    }

    return true;
}
```

Thanks

**Comment by YannikFirre** [comment]
Correction, the campaign remains "in progress" only if the class segment returns 0 or 1 subscriber.

Thanks a lot for this package, I launched 100 campaigns to 22k subscribers in one time, it worked like a charm ! :D

**Comment by ziming** [comment]
I got the Always in progress issue too.

**Comment by riasvdv** [comment]
Should be fixed in 2.11.6
